bldr.env.sample

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-# Only enable and set these if using RDS instead of a local PostgreSQL instance
+# Only enable and set these if you are using externally hosted PostgreSQL(RDS, Azure Database for PostgreSql etc).
+# set PG_EXT_ENABLED = true and change the PostgreSQL associated variables appropriatlely.
```

Comment:
I'd add some clarity here and state: If using an externally hosted PostgreSQL instance, set PG_EXT_ENABLED = true and also change PG_USER, PG_PASSWORD and POSTGRES_HOST.

Reply:
Resolved